yaml/com/ibm/Control/Host/PCIeLink.interface.yaml

```diff
@@ -0,0 +1,12 @@
+description: >
+    PCIe Link Interface is used to reset the PCIe link.
+
+properties:
+    - name: linkReset
+      type: boolean
+      default: false
+      description: >
+          When set to true resets the PCIe Link. The service
+          which monitors this property moves it back to the
+          default.
```

Comment:
"moves it back to false." is more clear. Any more detail about what the reset actually does?